tools/asm_module.py

```diff
@@ -2,7 +2,7 @@
 from __future__ import print_function
 import sys, re, itertools
 
-import shared, js_optimizer
+from . import shared, js_optimizer
```

Comment:
wait, then why does this work?

Reply:
Because `tools/asm_module.py` is always imported as a module by script and never directly runs as a script. (In other words, we never call `python tools/asm_module.py`.)

Modules support `from .` but scripts don't.

Reply:
Ok, thanks. Now I really do get it :)